docs/using-the-compiler.rst

```diff
@@ -372,6 +372,12 @@ Input Description
         {
           // Choose which model checker engine to use: all (default), bmc, chc, none.
           "engine": "chc",
+          // Chose which targets should be checked: all (default), constantCondition,
+          // underflow, overflow, divByZero, balance, assert, popEmptyArray.
+          // See the Formal Verification section for the targets description.
```

Comment:
@chriseth  Docs references don't work here, so just reference by text, not sure if this works